pkg/referrerstore/plugin/plugin_test.go

```diff
@@ -111,7 +110,7 @@ func TestPluginMain_GetBlobContent_InvokeExpected(t *testing.T) {
 	subject := common.Reference{
 		Original: "localhost",
 	}
-	result, err := storePlugin.GetBlobContent(context.Background(), subject, "", v1.Descriptor{})
+	result, err := storePlugin.GetBlobContent(context.Background(), subject, "")
```

Comment:
questions, is the storePlugin mocked out? curious if we were to setup integration tests, what the basic scenario to cover to cover oras changes like this.

Reply:
We do have mocks for the referrer store [here]() but to test ORAS changes we would probably need to set up a synthetic test where we have a test registry with a signed image where we can test each of the different ORAS store methods to see if it returns properly. I'm not sure how else we'd test the ORAS go library operations.